package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "electron-process-manager",
+  "name": "ssdc-electron-process-manager",
```

Comment:
Forgot? :)

Reply:
Was just to use it in local npm server, remove it from master branch